VISION.md

```diff
@@ -0,0 +1,41 @@
+### The Danger Vision
+
+Danger is a long term project, from our current viewpoint her role is
+
+ * Provide the best API for communication in Continuous Integration.
```

Comment:
That's super abstract, but probably the only way we can describe it so that it still works in a few years